src/WireMock.ts

```diff
@@ -46,42 +50,51 @@ export class WireMock {
         response: IWireMockResponse,
         features?: IWireMockFeatures,
     ): Promise<IMockedRequestResponse> {
-        const mockedRequest = createWireMockRequest(request, features);
-        const mockedResponse = createWireMockResponse(response, features);
+        let featuresToUse;
+        
+        if(this.featuresDefault){
```

Comment:
if you have the default, you might still want to override them with new ones. i will try to come up with some suggestions in a day or so